crates/hir_def/Cargo.toml

```diff
@@ -33,6 +33,8 @@ mbe = { path = "../mbe", version = "0.0.0" }
 cfg = { path = "../cfg", version = "0.0.0" }
 tt = { path = "../tt", version = "0.0.0" }
 
+backtrace = "0.3"
```

Comment:
ignore this. i've been using this to get some debugging information.

Reply:
I think you would like https://github.com/rust-analyzer/rust-analyzer/blob/bd675c8a8bdd3fda239bee3d3f31acd8679655b9/crates/stdx/src/lib.rs#L30

Reply:
oh that's nice!

Reply:
@cynecx Check twitter inbox